### PR TITLE
[CLNP-3623] chore: update sample

### PR DESCRIPTION
**Pull request by bang9**
ticket: [CLNP-3623]

[CLNP-3623]: https://sendbird.atlassian.net/browse/CLNP-3623?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/api.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  createUser: async (userId: string, nickname = '', profileUrl = ''): Promise<void> => {
+    const url = `https://api-${appId}.sendbird.com/v3/users`;
+    await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json; charset=utf8',
+        'Api-Token': apiToken,
+      },
+      body: JSON.stringify({ user_id: userId, nickname, profile_url: profileUrl }),
+    });
```
Comment (liamcho) [inline src/api.ts:32]: Better to Add this?
```
const data = await response.json();
    if (!data) throw new Error('Failed to xxx');
```

Comment (bang9) [reply src/api.ts:32]: Since it's a sample, excessive handling is not necessary. It is important to focus on the main logic

File: README.md [hunk 2]
```diff
@@ -47,9 +47,9 @@ Refer to the comments in the provided code for more information.
```
Comment (liamcho) [inline README.md:47]: What is embed example for?

**Review by chohongm — APPROVED** [review]
LGTM

File: src/api.ts (hunk 1 above)
Comment (liamcho) [reply src/api.ts:32]: ok

File: README.md (hunk 2 above)
Comment (bang9) [reply README.md:47]: It's a self-service

**Pull request merged**